odoo/models.py

```diff
@@ -896,7 +896,10 @@ def load(self, fields, data):
             cr.execute('ROLLBACK TO SAVEPOINT model_load')
             ids = False
 
-        if ids and self._context.get('defer_parent_store_computation'):
+        # bypass forced parent_store_compute to do it manually later
+        # by setting `manually` in `defer_parent_store_computation` key
+        defer_parent_compute = self._context.get('defer_parent_store_computation')
+        if ids and defer_parent_compute and defer_parent_compute != 'manually':
```

Comment:
That doesn't look like a good idea. Importing data in batches in separate workers may sound like a good idea performance-wise, but it's a bad idea transaction-wise.
By deferring the update of the hierarchy tree we introduce a limited inconsistency inside the import transaction itself, but the tree stays consistent outside of that transaction. Turning the auto-update permanently off is asking for trouble: concurrent transactions are going to see inconsistent data for the whole duration of the "import operations" (without knowing that it is inconsistent), and there is no guarantee that the tree will ever be correctly updated in the end.
I'd rather work on improving the speed of batch imports within a single transaction, e.g by offering an API to import multiple files at once.